src/metrics.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+from sklearn.metrics import mean_squared_error
+
+
+class MSEMetric:
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def evaluate_mse_class(y_pred: pd.DataFrame, y_true: pd.DataFrame) -> dict:
+        result = {}
+
+        for column in y_pred.columns:
+            result[column] = mean_squared_error(y_pred[column], y_true[column], squared=False)
+
+        return result
+
+    def evaluate(self, y_pred: pd.DataFrame, y_true: pd.DataFrame):
```

Comment:
Please notice, that the metric used in competition is not Mean MSE, but Mean RMSE: https://www.kaggle.com/competitions/feedback-prize-english-language-learning/overview/evaluation

Reply:
Here I set `squared=False` - by documentation this function returns RMSE